src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -174,10 +199,27 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         const fileVersionId = getProp(match, 'params.fileVersionId', currentFileVersionId);
 
         history.push(
-            activeAnnotationId ? `/activity/annotations/${fileVersionId}/${activeAnnotationId}` : '/activity/',
+            activeAnnotationId
+                ? `/activity/annotations/${fileVersionId}/${activeAnnotationId}`
```

Comment:
Seems like we may have this string duplicates in a few places, here and `handleDeeplinkedAnnotationRedirect`.